packages/client/src/components/documents/index.tsx

```diff
@@ -90,6 +100,34 @@ import { DocumentsFilters } from './ documents-filters.js';
   }
 `;
 
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions -- used by codegen
+/* GraphQL */ `
+  query SuggestDocumentMatchingTransactions($documentId: UUID!) {
+    suggestDocumentMatchingTransactions(documentId: $documentId) {
+      ... on CommonTransaction {
+        id
+        eventDate
+        sourceDescription
+        commonEffectiveDate: effectiveDate
+        amount {
+          formatted
+        }
+        chargeId
+      }
+      ... on ConversionTransaction {
+        id
+        eventDate
+        sourceDescription
+        conversionEffectiveDate: effectiveDate
+        amount {
+          formatted
+        }
+        chargeId
+      }
```

Comment:
Those attributes are part of the Transaction interface, so no need to spread types here

```suggestion
      id
      eventDate
      sourceDescription
      conversionEffectiveDate: effectiveDate
      amount {
        formatted
      }
      chargeId
```